### PR TITLE
Lesson1 updates

### PR DESCRIPTION
**Pull request by sdoward**
Changes repl references with codeboard

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -17,8 +17,8 @@ title: Intro to Java
 description: This is the homepage of the "Intro to Java" course at ReDI School
 footer_content: "Made with ❤ by teachers at <a href=\"https://www.redi-school.org\">ReDI School</a>."
 aux_links:
-  "Code editor at repl.it":
-    - "https://repl.it/"
+  "Code editor at codebranch.io":
```
Comment (julia-altenbuchner) [inline _config.yml:20]: What's codebranch? Do you mean codeboard.io?

File: lesson1/exercise1.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ Write them down/remember them to share later with the class.
 {:toc}
 
 ## Precondition
-You should have opened the [repl.it](https://repl.it/) website and created a new *Java* repl (do not yet change things).
+You should have opened the [codeboard.io](https://codeboard.io/) website and created a new *Java* codeboard (do not yet change things).
 
-If not, please visit [this page](replit) to find out how and come back here after that.
+If not, please visit [this page](https://codeboard.io/) to find out how and come back here after that.
```
Comment (julia-altenbuchner) [inline lesson1/exercise1.md:19]: I don't understand the point of this sentence. It just links to the same page as the sentence above, no?

File: lesson1/exercise1.md [hunk 3]
```diff
@@ -14,9 +14,9 @@ Write them down/remember them to share later with the class.
 {:toc}
 
 ## Precondition
-You should have opened the [repl.it](https://repl.it/) website and created a new *Java* repl (do not yet change things).
+You should have opened the [codeboard.io](https://codeboard.io/) website and created a new *Java* codeboard (do not yet change things).
```
Comment (julia-altenbuchner) [inline lesson1/exercise1.md:17]: Maybe from here you can link to the instructions page you wrote

**Review by julia-altenbuchner — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 1 above)
Comment (sdoward) [reply _config.yml:20]: I always say code branch for some reason 🙈

File: lesson1/exercise1.md (hunk 2 above)
Comment (sdoward) [reply lesson1/exercise1.md]: Yes, removed

File: lesson1/codeboard.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: codeboard.io - a web-based Java editor
+parent: 1 - First Steps with Java
+nav_order: 7
+---
+
+# Write Java code with codebranch.io
```
Comment (jonas-halbach) [inline lesson1/codeboard.md:7]: codeboard instead of codebranch?

File: lesson1/exercise1.md [hunk 5]
```diff
@@ -14,9 +14,9 @@ Write them down/remember them to share later with the class.
...
 
-If not, please visit [this page](replit) to find out how and come back here after that.
+You can fine the setup instructions [here](codeboard.md)
```
Comment (jonas-halbach) [inline lesson1/exercise1.md:19]: fin**d** instead of fin**e**?

**Pull request merged**